src/localization.py

```diff
@@ -15,4 +13,7 @@ def get_localized(key):
 
     json_path = os.path.join(base_path, 'Languages', f'{language}.json')
     with open(json_path, 'r') as file:
-        return json.load(file)[key]
\ No newline at end of file
+        try :
+            return json.load(file)[key]
+        except :
+            return ""
```

Comment:
I would suggest to use english version here instead of empty string. That's described in #59.